### ISSUE TITLE
Unable to play Spotify songs in their entirety on my website

### ISSUE THREAD

**Issue by nakhla91** [spotify/web-api#1430]
Issue found on January 14th 2020.

Hello, I've created a website to share my Spotify playlists and I am only able to play 30 second previews of each song in my playlist. In order to play the full song, I am prompted to open the Spotify app. Is there a way to play the songs fully directly on my website? Thank you.

#### Expected behavior:
A user should be able to listen to a song in its entirety when embedding a Spotify playlist on my website.

As an example, please see the below blog where entire songs can be played.

https://urbanhelmetsusa.com/blogs/music/cali-cruising-music-playlist

#### Actual behavior:
 On my site (URL below) I am only able to play 30 second samples of each song in my playlist.

https://nakhla91.wixsite.com/mysite

**Comment by 0xNF** [comment]
Nope. The only way to play music is via the player API, and even that will only send music streaming commands to an open Spotify client (like the phone app, the desktop client, a registered smart speaker, etc).

**Comment by nakhla91** [comment]
So how is the website in the first link I sent able to play songs in their entirety while my own website is not?
